src/credentials/sso/diskCache.ts

```diff
@@ -4,80 +4,83 @@
  */
 
 import * as fs from 'fs'
-import * as crypto from 'crypto'
 import { SsoCache } from './ssoCache'
-import { ClientRegistration } from './clientRegistration'
-import { join } from 'path'
+import { SsoClientRegistration } from './ssoClientRegistration'
+import { join as joinPath } from 'path'
 import { homedir } from 'os'
-import { AccessToken } from './accessToken'
+import { SsoAccessToken } from './ssoAccessToken'
+import { getSHA1StringHash } from '../../shared/utilities/textUtilities'
 
 export class DiskCache implements SsoCache {
-    private cacheDir: string = join(homedir(), '.aws', 'sso', 'cache')
+    private cacheDir: string = joinPath(homedir(), '.aws', 'sso', 'cache')
+    // Treat the token or client registration as expired if within 15 minutes of expiration.
+    private TIME_ADDED_BEFORE_EXPIRY = 900000
```

Comment:
Consider a descriptive name that also indicates the units it represents (eg: TokenExpirationBufferMinutes, except this might represent milliseconds or some other unit)

Reply:
Changed to TOKEN_EXPIRATION_BUFFER_MS